### PR TITLE
Removes EMP blast from singularities, buffs radiation damage

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
[BALANCE] [QOL]
## About the PR 
This pr removes the emp blast from the singularity and buffs the radiation damage from singularities. This should lead to engineers not being forced to manage the singularity or do repairs, considering EMP blasts turn off your headset, and there is no way to counter EMP blasts. The radiation damage has been doubled, dealing approximately 30 toxic damage total. Stun is buffed per kyles request

## Why's this needed? 
To explain the reasoning behind this, the EMPs effects need to be fully explained. The EMP has 3 main effects, 1. It damages borgs. 2. It turns off/drains power from electronics (like sec gear and headsets) 3. It emags bots. The first one adds nothing, borgs already have no reason to be in the singularity chamber, this effect doesnt seem to add anything. The second effect forces engineers into a mindset of number go up or focus on maintaining the station. If you wish to increase power for a singularity, you will likely spend time nearby a singularity, which will lead to your headset getting off very frequently. The third one is the only interesting effect of the EMP blast, however I have not seen this utilized once. All the emp burst does is make the experience worse, with no counters. The radiation damage previously was very easy to ignore, this change should make wearing a radsuit if you wish to stay inside of inner engineering for extended periods of time a practical requirement. Oh I guess emp blasts can open doors too, eh, they can also turn off apcs which also eh, smallulos in public areas are fun and making it safer to do that seems like itll lead to fun scenarios.

## Changelog

```changelog
(u)Ikea
(*)EMP blast from the singularity has been removed, radiation damage has been buffed from singularities. Wear radsuits people!
```

### CODE REVIEW

**Comment by MatheusIsNotAvailable** [comment]
From what I am getting, the only non fun part of the EMP blast was it turning off headsets, why not just remove that aspect of it then? I always thought the singulo opening doors was fun, that is why people weld those and that is a fun thing to learn.

**Comment by ThatFiveGuys** [comment]
I dont want to nerf emps, not to mention if a borg wishes to help out with the singularity, them just being killed for even standing in the chamber isnt what Id define as fun.

**Comment by Tarmunora** [comment]
Isn't the EMPing what drains the field generators?

**Comment by ThatFiveGuys** [comment]
Just something field gens do passively, seen here: https://github.com/goonstation/goonstation/blob/master/code/obj/machinery/singularity.dm#L531

File: code/obj/machinery/singularity.dm [hunk 1]
```diff
@@ -167,8 +167,6 @@ for some reason I brought it back and tried to clean it up a bit and I regret ev
 			maxradius = INFINITY
 
 
-/obj/machinery/the_singularity/emp_act()
-	return // No action required this should be the one doing the EMPing
```
Comment (Tarmunora) [inline code/obj/machinery/singularity.dm:171]: These lines should probably stay

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/obj/machinery/singularity.dm (hunk 1 above)
Comment (ThatFiveGuys) [reply code/obj/machinery/singularity.dm]: Done!

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**